arrows/core/detected_object_set_input_csv.cxx

```diff
@@ -137,19 +182,42 @@ read_set( kwiver::vital::detected_object_set_sptr & set, std::string& image_name
   if ( d->m_first )
   {
     d->m_first = false;
-
     if ( ! d->get_input() )
     {
       return false; // indicate end of file.
     }
-
+    //This is what creates the map from file name to object set
+    //In the future maybe it should be computed if ever any 
+    if ( !image_name.empty() ){
```

Comment:
very minor, but prefer if brackets start on a new line, especially since it's consistent with the rest of the file.